pkg/logs/pipeline/pipeline.go

```diff
@@ -91,14 +91,14 @@ func (p *Pipeline) Flush(ctx context.Context) {
 	p.processor.Flush(ctx) // flush messages in the processor into the sender
 }
 
-func getDestinations(endpoints *config.Endpoints, destinationsContext *client.DestinationsContext, pipelineID int) *client.Destinations {
+func getDestinations(endpoints *config.Endpoints, destinationsContext *client.DestinationsContext, pipelineID int, serverless bool) *client.Destinations {
```

Comment:
We should really be using build tags to determine when we're in serverless. That will allow us to compile out whole sections of code that are unreachable to us and thus could decrease the size of the extension binary.

Reply:
Should we create another serverless ticket for that?

Reply:
added here https://datadoghq.atlassian.net/browse/SVLS-3997